certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3443 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.4",
+    "title": "Virus Total APIV3",
+    "description": "user will be able to enter an ip address and get the exact experience of a report in virus total on this IP.",
+    "x-ms-api-annotation": {
+      "status": "Preview"
+    }
+  },
+  "host": "www.virustotal.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+
+  "paths": {
+    "/api/v3/ip_addresses/{ip}": {
+      "get": {
+        "summary": "Ip scan report V3",
+        "description": "scan Ip to get full report",
+        "operationId": "VirusTotalGetIpScanV3",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "ip",
+            "description": "Ip to verify",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "IP Address",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/ipResult"
+            }
+          }
+        }
+      }
+    },
+    "/api/v3/urls": {
+      "post": {
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "summary": "Analyse an URL",
+        "description": "Analyse an URL",
+        "operationId": "VirusTotalAnalysisurlV3",
+        "parameters": [
+          {
+            "in": "formData",
+            "name": "url",
+            "description": "URL to verify",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "URL"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/urlReport"
+            }
+          }
+        }
+      }
+    },
+    "/api/v3/analyses/{id}": {
+      "get": {
+        "summary": "Retrieve information about a file or URL analysis",
+        "description": "Retrieve information about a file or URL analysis. This action uses the output id of Analyse an URL or Upload and analyse a file.",
+        "operationId": "VirusTotalRetrieveInfo",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "report",
```

Comment:
Description should begin with upper case letter. Why `report` is more appropriate than `Analysis identifier` as in https://developers.virustotal.com/v3.0/reference#analysis?

Reply:
Done